shared/wallets/wallet-list/wallet-row/index.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import * as React from 'react'
+import {Box2, ClickableBox, Icon, Text, Avatar} from '../../../common-adapters'
+import {
+  globalStyles,
+  globalMargins,
+  globalColors,
+  isMobile,
+  platformStyles,
+  collapseStyles,
+} from '../../../styles'
+
+type Props = {
+  isSelected: boolean,
+  name: string,
+  keybaseUser: string,
+  contents: string,
+  onSelect: () => void,
+}
+
+const rightColumnStyle = platformStyles({
+  isElectron: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  },
+})
+
+const WalletRow = (props: Props) => {
+  const backgroundColor = props.isSelected ? globalColors.blue : globalColors.white
+
+  const titleStyle = collapseStyles([
+    rightColumnStyle,
+    {
```

Comment:
we need to use styleSheetCreate and not inline this stuff

Reply:
Done, except for Icon's style, which doesn't play well with styleSheetCreate